src/seedu/addressbook/data/person/Person.java

```diff
@@ -15,6 +15,16 @@
     private Email email;
     private Address address;
 
+    /**
+     * Unique to Person object at when first instantiated.
+     */
+    private int sequenceNumber;
+
+    /**
+     * Keep track of next sequence number to assign for new Person object.
+     */
+    private static int nextSequenceNumber = 0;
```

Comment:
Would it be better to initialize `nextSequenceNumber` to 1 rather than increment it in the constructor?